py-utils/src/utils/msg_bus/kafka_python_example.py

```diff
@@ -0,0 +1,55 @@
+from __future__ import absolute_import
```

Comment:
Please move it to py-utils/test/message_bus/

Reply:
Moved